contrib/nushell/patches/autoload.patch

```diff
@@ -0,0 +1,228 @@
+From f241110005006f10e16c6080ebab7abec6f3e5ce Mon Sep 17 00:00:00 2001
+From: Darren Schroeder <343840+fdncred@users.noreply.github.com>
+Date: Tue, 25 Jun 2024 20:31:54 -0500
+Subject: [PATCH] implement autoloading (#13217)
+MIME-Version: 1.0
+Content-Type: text/plain; charset=UTF-8
+Content-Transfer-Encoding: 8bit
+
+# Description
```

Comment:
the pr template here is really annoying me but idk

Reply:
If you want me to drop it I'll do that, I just fetched the patch from github as is

Reply:
it's ok